en/snap/with-plugins.md

```diff
@@ -56,269 +62,308 @@ Here is the list of Content Management System plugins and extensions that are su
+## WordPress - WooCommerce
+Midtrans ❤️ WooCommerce! This plugin allows secure online payment on your WooCommerce store, without ever needing your customer to leave your WooCommerce store! It has a beautiful built-in responsive payment interface. Midtrans strives to make payments simple for you and your customers. It supports various online payment channels. Midtrans supports WooCommerce v2 and v3.
 
-Our WooCommerce plugins also available on [Wordpress plugins store](https://wordpress.org/plugins/midtrans-woocommerce/). If it's not listed there, you can always download and [install as described below](#b-manual-installation).
+Midtrans-WooCommerce plugin is also available on [WordPress plugins store](https://wordpress.org/plugins/midtrans-woocommerce/). If you cannot find it listed there, you can always download and install it manually. For more details, refer to Manual Installation.
 
-#### Requirements:
-* WordPress v3.9 or greater **|** Tested up to v5.x
-* WooCommerce v2 or greater **|** Tested up to v3.5.2
-* PHP version v5.4 or greater
-* MySQL version v5.0 or greater
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* WordPress v3.9 or later **|** Tested up to v5.x
+* WooCommerce v2 or later **|** Tested up to v3.5.2
+* PHP version v5.4 or later
+* MySQL version v5.0 or later
 * PHP CURL enabled server/host
+* Midtrans plugin for WooCommerce [ [GitHub](https://github.com/veritrans/SNAP-Woocommerce) | [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip) ]
+
+#### WooCommerce Plugin Installation
+Select **any one** of the installation options given below.
+
+**Simple Installation**
+
+To install Midtrans-WooCommerce plugin, follow the steps given below.
+  1. Login to your WordPress administration panel.
+
+  2. Go to **Plugins** menu.
 
-#### Installation:
+  3. Click **add new**.
 
-Choose **one** from installation options **A or B**:
+  4. Search for **Midtrans-WooCommerce plugin**.
 
-#### A. Simple Installation
-   1. Login to your Wordpress admin panel.
-   2. Go to Plugins menu, click add new. Search for Midtrans-WooCommerce plugin.
-   3. Install and follow on screen instructions.
-   4. Proceed to Configuration Process.
+  5. Click **Install Now** and follow on-screen instructions.
 
-If you are unable to install using this method, please proceed with [manual installation below](#b-manual-installation).
+The plugin is installed successfully. Proceed to [WooCommerce Plugin Configuration](#wooCommerce-plugin-configuration).
+
+If you are unable to install, proceed to Manual Installation.
+
+**Manual Installation**
+
+To install Midtrans-WooCommerce plugin manually, follow the steps given below.
+   1. Download the plugin file from the link given above.
 
-#### B. Manual Installation
```

Comment:
In the file `snap/overview.md` you are leaving the `A.` `B.` in the list that is meant to be choice. Why do you remove it here? which consistent approach you want to take?